### DISCUSSION TITLE
Org Parsing Library

### DISCUSSION THREAD

**Discussion by jethrokuan** [org-roam/org-roam-ui#2] [v0]
If you intend to do something with the content of the Org files (e.g. a viewer), I think using @rasendubi's Uniorg is a good idea, as he'd built it to support publishing Org-roam files.

https://github.com/rasendubi/uniorg/

**Comment by tefkah** [comment]
Ah that's great! We were thinking of doing something like that, do you perhaps know what the difference is between this and https://github.com/orgapp/orgajs? I couldn't get this to work yet, but it seems a bit more mature than uniorg.

**Reply by jethrokuan** [reply]
Orgajs does look a bit more mature, let me ask the uniorg author for his opinion.
